src/SDKs/Peering/Management.Peering.Tests/Peering.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Peering.Tests</PackageId>
+    <AssemblyName>Peering.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+    <Description>Test Project for Peering tests</Description>
```

Comment:
Please rename the directory `Management.Peering.Tests` to `Peering.Tests`

Reply:
in the latest the name was changed to Peering.Tests.